modules/op-installing-pipelines-operator-using-the-cli.adoc

```diff
@@ -2,15 +2,14 @@
 // * install_config/installing-pipelines.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="op-installing-pipelines-operator-using-the-cli_{context}"]
-= Installing the {pipelines-shortname} Operator using the CLI
+[id="op-installing-pipelines-operator-by-using-the-cli_{context}"]
```

Comment:
```suggestion
[id="op-installing-pipelines-operator-using-the-cli_{context}"]
```
I suggest keeping this as it was for this reason: ID needs to match the name of the file, and that would require additional change around, changing links etc. But the heading can slightly differ from the ID, so I would suggest keeping the new heading, but leaving the ID as it was before